deployment/helm/README.md

```diff
@@ -100,5 +102,31 @@ To build and run the chart locally, follow the following steps
     ```
 1. Build the chart: 
     ```shell script
-    helm package
-    ```
\ No newline at end of file
+    helm package .
+    ```
+
+## Installing the chart
+
+Installing the chart locally with the following command:
+
+```shell script
+helm install postfacto <local-chart.tgz>
+```
+
+Will automatically install `postgresql` with persistence enabled, which means the configuration and data
+survive `helm uninstall`.
+
+So the next time `postfacto` gets installed, `postgresql` will attempt to generate a new password and fail to authenticate.
+
+As of now there are two solutions to this problem:
```

Comment:
These feel like instructions for a `helm upgrade` to me. Generally when I `helm uninstall && helm install` I want to start from a clean slate so I would delete the PVC manually after then uninstall

Reply:
True you could delete them manually but only if you know they exist. Unfortunately it took me a while before noticing the pvc because when you run `kubectl get all` they are not part of the resources in the output. This is I think a bug on the posgresql side as I tested it in isolation and I got the same problem.

Reply:
I've updated the doc which now only describes how to cleanup the state using kubectl.